specification/components/schemas/eRS-ChangeShortlistAndSendForTriage-Parameters.yaml

```diff
@@ -0,0 +1,38 @@
+title: eRS-ChangeShortlistAndSendForTriage-Parameters
+type: object
+description: "Parameters provided when looking to change the services a referral has currently shortlisted"
+required:
+  - resourceType
+  - meta
+  - parameter
+properties:
+  resourceType:
+    type: string
+    enum:
+      - 'Parameters'
+    example: Parameters
+  meta:
+    type: object
+    required:
+      - profile
+    properties:
+      profile:
+        type: array
+        items:
+          type: string
+          enum:
+            - "https://fhir.nhs.uk/STU3/StructureDefinition/eRS-ChangeShortlistAndSendForTriage-Parameters-1"
+          example: "https://fhir.nhs.uk/STU3/StructureDefinition/eRS-ChangeShortlistAndSendForTriage-Parameters-1"
+  parameter:
+    type: array
+    description: |
+      | Field                             | Cardinality | Notes                                                                                    |
+      | --------------------------------- | ----------- | ---------------------------------------------------------------------------------------- |
+      | `Shortlist`                       | `1..1`      | Shortlist indicating the new services a referral should have shortlisted                 |
```

Comment:
Might be worth highlighting here that only the shortlist should only contain a single service.